src/pages/EntityDetails/Model/Model.tsx

```diff
@@ -242,6 +243,7 @@ const Model = () => {
           </>
         )}
         {shouldShow("logs", query.activeView) && <Logs />}
+        {shouldShow("secrets", query.activeView) && <Secrets />}
```

Comment:
I think we might need to add `case "secrets":` in the definition of `shouldShow`. Currently, the `Secrets` page doesn't render.

Reply:
Oops, thanks!